### DISCUSSION TITLE
Public lists

### DISCUSSION THREAD

**Discussion by AngelOnFira** [community/community#8607] [Other Feature Feedback, Questions, & Ideas] [answered]
I'd love the ability to make lists public. I have a list of the projects my friends are working on, and it would be cool to show that off!

![image](https://user-images.githubusercontent.com/14791619/145443619-23f19ae3-345c-4f43-9ab0-e69b6be72d09.png)

**Comment by brianlovin** [comment, marked as answer]
Hey @AngelOnFira right now lists are public by default. In the future, we will explore adding the ability to create _private_ lists, but for now you should be good to go!

**Reply by AngelOnFira** [reply]
Thanks Brian, I originally thought they weren't since I couldn't see it when I opened Github in a private browser. But my friends are able to see it, so all good!

**Reply by dantehemerson** [reply]
@brianlovin Do you mean *private by default*? Because I can't see my lists from another profile.

**Reply by warengonzaga** [reply]
Where to check it? I can't see it on my profile.

**Reply by m1ga** [reply]
@WarenGonzaga  https://github.com/WarenGonzaga?tab=stars under `Lists` at the top

**Reply by warengonzaga** [reply]
@m1ga I know, when I see my profile as a guest user I can't see it.
![image](https://user-images.githubusercontent.com/15052701/145718205-8635e2a1-f487-471e-8783-a335186cc251.png)

**Reply by m1ga** [reply]
Think the definition of private/public is not covering users without an account. Meaning that I can see your list when I'm logged in, so the list is not just private for you but public for every github user.

**Reply by warengonzaga** [reply]
Oh, meaning the list is only viewable for logged-in GitHub users? @m1ga

**Reply by m1ga** [reply]
![Screenshot_20211212_162611](https://user-images.githubusercontent.com/4334997/145718424-0d3bfd1b-976d-4985-9078-48472577679a.png)

**Reply by warengonzaga** [reply]
Thanks for the help @m1ga! Now I understand.

**Reply by brianlovin** [reply]
This will be fixed soon so that logged-out users will be able to see your lists.

**Comment by feynon** [comment]
I can't see the link publicly as well, would love to have the public sharing feature by default.

**Reply by brianlovin** [reply]
We will make sure all lists are visible to logged-out users soon. Lists are public by default for now, and in the future we'll look into adding private options 😊

**Comment by sudofox** [comment]
I'd like to be able to know whether my lists are visible or not. Sometimes I want to bookmark a repository without it being seen by anyone else.

**Reply by sudofox** [reply]
It's also not clearly communicated that adding something you a list stars it

**Reply by brianlovin** [reply]
Thanks for the note — yes, all lists are visible to the public right now, and adding something to a list will also star it. We're looking into more options in the future to have stars + lists be separate, and to have private list options.

**Reply by AbhimanyuAryan** [reply]
@brianlovin please make them private :(

**Reply by brianlovin** [reply]
We will explore options for private lists in the future, but for now they will all be public. The contents of a list, like private repositories, will only appear to people who have permission to see those repositories.

**Comment by relliv** [comment]
I loved this feature but I did not find way to share as public list until @m1ga's message. Yes actually listing public but list URL should be improve. Currently listing as `github.com/stars/username/lists` but `stars` part is not typing friendly. I think this way more clear `github.com/username/lists`.
